### DISCUSSION TITLE
Right Clic on Text Editors clears Text Selection

### DISCUSSION THREAD

**Discussion by abmace** [dbeaver/dbeaver#18154] [General]
In the text editor if you have selected text and right-click on it, just to make secondary menu to pop, to select copy; the text gets deselected. This is a nuance or an inconvenience more than a crtical error, but nonetheless I felt compelled to report such thing though.

**Comment by joshcsmith13** [comment]
I HATE this new "feature".  If you want to add "selected query" contextual menu entries, then great, but don't eliminate the traditional "Copy" command!!!
This is specifically a pain when I just want to copy a critical value for one field from my where clause, and the only options I have with my mouse are now for the entire query.

**Comment by E1izabeth** [comment]
@abmace thank you for reporting. The fix will be available in the next release.
